### PR TITLE
Setup for different mailing list for new users/deactivations

### LINKED ISSUE #986: Setup for different mailing list for new users/deactivations

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #986

## PR includes

-   [x] Feature

### CODE REVIEW

File: src/components/MailingLists/MailingLists.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { trpc } from '../../trpc/trpcClient';
+import { getOrgUnitTitle } from '../../utils/organizationUnit';
+
+import { tr } from './MailingLists.i18n';
+
+export const MailingLists = () => {
+    const organizationUnitQuery = trpc.organizationUnit.getList.useQuery({ search: '' });
+
+    const [orgUntId, setOrgUnitId] = useState<string | undefined>(undefined);
+    useEffect(() => {
```
Comment (tsumo) [inline src/components/MailingLists/MailingLists.tsx:16]: What this effect is doing?

File: src/modules/userSchemas.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ export const getUserListSchema = z.object({
     active: z.boolean().optional(),
     cursor: z.string().nullish(),
     mailingSettings: z.enum(mailingSettingType).optional(),
```
Comment (tsumo) [inline src/modules/userSchemas.ts:50]: shoud be `mailingSettings: {type: ..., orgId: ...}.optional()` because they only used together

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/MailingLists/MailingLists.tsx (hunk 1 above)
Comment (nyet-ty) [reply src/components/MailingLists/MailingLists.tsx:16]: automatically set first organization id to selected tab

File: src/components/MailingLists.tsx [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
-                <MailingList key={type} mailingSettings={type} />
+                <MailingList key={type} mailingSettings={type} organizationUnitId={orgUntId} />
             ))}
         </AdminPanelLayout>
     );
 };
+
+export const MailingLists = () => {
+    const organizationUnitQuery = trpc.organizationUnit.getList.useQuery({ search: '' });
+    if (!organizationUnitQuery.data?.length) return null;
```
Comment (tsumo) [inline src/components/MailingLists.tsx:40]: Maybe add a message `No organzations` on length = 0

**Review by tsumo — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**